package.json

```diff
@@ -24,9 +24,9 @@
 		"publish": "electron-forge publish",
 		"lint": "eslint --ext .ts,.tsx,.js,.jsx,.mjs .",
 		"format": "prettier . --write",
-		"test": "jest",
-		"test:watch": "jest --watch",
-		"e2e": "npx playwright install && npx playwright test",
+		"test": "NODE_ENV=test jest",
+		"test:watch": "NODE_ENV=test jest --watch",
```

Comment:
From reviewing [documentation](https://jestjs.io/docs/getting-started#using-babel) and testing locally, defining this explicitly is unnecessary as Jest sets `NODE_ENV=test` if it is not set to something else. 

> Jest will set process.env.NODE_ENV to 'test' if it's not set to something else.

That said, I'm fine keeping this here to be explicit, if preferred.

Reply:
> I see we use process.env.E2E in some other places. Would it make sense to unify this to use one or another everywhere?
> From reviewing [documentation](https://jestjs.io/docs/getting-started#using-babel) and testing locally, defining this explicitly is unnecessary as Jest sets NODE_ENV=test if it is not set to something else.

Good points, both. I think it makes sense to leverage the existing `process.env.E2E` event for E2E tests -- I hadn't yet noted that we were already using it. I'll update both areas and check that the env is being set prior to running E2E builds, as noted in https://github.com/Automattic/studio/pull/140#discussion_r1605016916.